src/utils/sort-versions.ts

```diff
@@ -33,14 +61,15 @@ function compare(a: semver.SemVer | string, b: semver.SemVer | string) {
  * @returns {RunnableVersion[]}
  */
 export function sortVersions(versions: RunnableVersion[]): RunnableVersion[] {
-  type VerSemRun = [
-    ver: string,
-    sem: semver.SemVer | null,
-    run: RunnableVersion,
-  ];
-  const sorted = versions
-    .map((run): VerSemRun => [run.version, semver.parse(run.version), run])
-    .sort(([vera, sema], [verb, semb]) => compare(sema || vera, semb || verb));
-  sorted.forEach(([_1, _2, run], idx) => (versions[idx] = run));
-  return versions;
+  type VerSemRun = [ver: string, run: RunnableVersion];
+
+  const filtered = versions.filter((runnable) =>
+    semver.parse(runnable.version),
+  );
+  const sorted = filtered
+    .map((run): VerSemRun => [run.version, run])
+    .sort(([vera], [verb]) => -semverCompare(vera, verb));
+  sorted.forEach(([_1, run], idx) => (filtered[idx] = run));
+
+  return filtered;
```

Comment:
keep old implementation but replace

```diff
-    .sort(([vera, sema], [verb, semb]) => compare(sema || vera, semb || verb));
+    .sort(([vera, sema], [verb, semb]) => -semverCompare(sema || vera, semb || verb));
```

Reply:
hmm - isn't this what it is now? not sure what i need to replace since i'm already using `-semverCompare` - would you mind clarifying?